src/integration/pages/mediaAssetPage/persian/__snapshots__/amp.test.js.snap

```diff
@@ -74,6 +74,19 @@ Object {
 }
 `;
 
+exports[`AMP Media Asset Page Analytics ATI 1`] = `
+Object {
+  "vars": Object {
+    "contentType": "article-media-asset",
+    "domain": "test.bbc.co.uk",
+    "sections": "Persian, Persian - صفحه ایران, Persian - MAP, Persian - صفحه ایران - MAP, Persian - undefined-category",
```

Comment:
is `undefined-category` expected here?

Reply:
It looks like a bug with TC2 assets - the CPS one has correct category. I'll raise it in Bug Triage, and we can decide whether we want to fix it or not.

Reply:
Sorry, it appears to be both TC2 & CPS MAPs (but not all). Either way, we can review as a team and see if it's worth fixing.